Makefile

```diff
@@ -80,3 +76,34 @@ bin/packr2-${PACKR_VERSION}:
 .PHONY: uibundle
 uibundle: bin/packr2
 	GO111MODULE=on cd cmd/cloudinfo && $(abspath bin/packr2)
+
+.PHONY: generate-openapi
+generate-openapi: validate-openapi ## Generate go server based on openapi description
```

Comment:
This generate-openapi make target seems to be unnecessary.

Reply:
thx, deleted it